src/interpreter/token.rs

```diff
@@ -0,0 +1,8 @@
+// split to iterator.
+use std::str;
+
+type MyIter<'a> = str::SplitWhitespace<'a>; //type aliasing
+
+pub fn token(text: &str) -> MyIter {
+    text.split_whitespace()
```

Comment:
add return for consistency

Reply:
I would say we should remove returns in other places for consistency so as to conform to standard Rust style.